staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go

```diff
@@ -32,7 +32,7 @@ func Kind(kind string) schema.GroupKind {
 	return SchemeGroupVersion.WithKind(kind).GroupKind()
 }
 
```

Comment:
Please also remove `Group` in L30.

Address this in other files as well.

Reply:
Please take a look again, , Thank you!